src/cmd-build

```diff
@@ -61,6 +65,23 @@ prepare_build
 ostree --version
 rpm-ostree --version
 
+# RFC3339 formatter
+RFC3339="%Y-%m-%dT%H:%M:%SZ"
+
+if [ "${TIMESTAMP}" == 1 ]; then
+    if [ ! -e "${workdir:?}"/builds/builds.json ]; then
+        fatal "No builds.json file found; unable to apply timestamp"
+    fi
+    builds_json="${workdir}"/builds/builds.json
```

Comment:
nit: this would be slightly more compact if you moved this definition before the existence check (and then used the `:?` form in the definition).  Then you could use the DRYer `if [ ! -f "${builds_json}" ]` in the existence check.